akamai/edgegrid/edgegrid.py

```diff
@@ -36,7 +36,7 @@
     from urllib.parse import urlparse, parse_qsl, urlunparse
 else:
```

Comment:
This bit of the code will need to be reworked. There's no reason to leave it this way if we're removing Python 2 support.